Mathlib/Data/Finset/Basic.lean

```diff
@@ -1174,14 +1174,14 @@ theorem ssubset_insert (h : a ∉ s) : s ⊂ insert a s :=
 #align finset.ssubset_insert Finset.ssubset_insert
 
 @[elab_as_elim]
-theorem cons_induction {α : Type _} {p : Finset α → Prop} (h₁ : p ∅)
-    (h₂ : ∀ ⦃a : α⦄ {s : Finset α} (h : a ∉ s), p s → p (cons a s h)) : ∀ s, p s
+theorem cons_induction {α : Type _} {p : Finset α → Prop} (base : p ∅)
+    (ind : ∀ ⦃a : α⦄ {s : Finset α} (h : a ∉ s), p s → p (cons a s h)) : ∀ s, p s
```

Comment:
Wouldn't `empty` and `cons` be better names here?

Reply:
I imagine we want to give `induction` a few lines below the same treatment.

Reply:
I tried those first! But it caused a name clash with the normal `Finset.cons` in the body of `cons_induction`. Nothing that can't be resolved with the appropriate qualifiers but it started to smell a bit fishy. Would you like me to go ahead with that?